### PR TITLE
[ADD] website_baidu_map_multi

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

**Comment by noahzaozao** [comment]
LGTM
@elicoidal @seb-elico  Please review this PR , thanks.

File: website_baidu_map_multi/LICENSE [hunk 1]
```diff
@@ -0,0 +1,661 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (elicoidal) [inline website_baidu_map_multi/LICENSE]: remove this file

File: website_baidu_map_multi/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+website_baidu_map_multi
```
Comment (elicoidal) [inline website_baidu_map_multi/README.rst]: one line before (same length)
Name should be Website Baidu Map (Multi location)

File: website_baidu_map_multi/README.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+To configure this module, you need to:
+
+ * No specific configuration needed.
+
+Usage
+=====
+
+
+For further information, please visit:
```
Comment (elicoidal) [inline website_baidu_map_multi/README.rst]: remove

File: website_baidu_map_multi/README.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (elicoidal) [inline website_baidu_map_multi/README.rst]: remove if empty

File: website_baidu_map_multi/README.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Contributors
+------------
+
+* Liu Lixia <liu.lixia@elico-corp.com>
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
```
Comment (elicoidal) [inline website_baidu_map_multi/README.rst]: https (x3

File: website_baidu_map_multi/README.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+Credits
+=======
+
+
+Contributors
+------------
+
+* Liu Lixia <liu.lixia@elico-corp.com>
+
+Maintainer
```
Comment (elicoidal) [inline website_baidu_map_multi/README.rst:31]: Should be elico corp footer, not the OCA one

File: website_baidu_map_multi/views/company_shop_view.xml [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+                    <field name="company_id" invisible="1"/>
+                </tree>
+            </field>
+        </record>
+
+      <template id="baidu_map_muti" name="map_multi" inherit_id="website.company_description" customize_show="True">
+          <xpath expr="//div[@id='allmap']" position="replace">
+            <div id="allmap">
+                <script type="text/javascript">
+                    // 百度地图API功能
```
Comment (elicoidal) [inline website_baidu_map_multi/views/company_shop_view.xml]: remove dead code

**Comment by xie8899** [comment]
@noahzaozao @elicoidal Done

**Comment by elicoidal** [comment]
some mismatch here: project long term?

**Comment by victor-ditay** [comment]
Two modules in one PR?
`gap_analysis_project_long_term` `website_baidu_map_multi`

This is an exception @elicoidal @seb-elico or maybe a mistake? Since the name of PR is 8.0-add-website_baidu_map_multi_from_superman I think should be added `website_baidu_map_multi`

**Comment by noahzaozao** [comment]
@victormartinelicocorp Good catch , I didn't find it as well , I removed it and now I think it's LGTM.

**Comment by seb-elico** [comment]
LGTM

**Pull request merged**